tests/e2e/support/cta/files/publicLink.ts

```diff
@@ -0,0 +1,25 @@
+import { DataTable } from '@cucumber/cucumber'
+import { PublicLinkPage } from '../../page/publicLinkPage'
+import { filesCta } from '../../cta'
+
+export const checkResourcesExistence = async ({
+  actionType,
+  stepTable,
+  resourceType = 'public'
```

Comment:
```suggestion
  pageType = 'public'
```

Reply:
resourceType is not suitable for that case. Better to change to a more suitable name